cypress/page/products-list.page.ts

```diff
@@ -3,8 +3,10 @@ class ProductsList {
     private proceedToCheckoutButton: string;
 
     constructor(){
-        this.addToChartButton = "#center_column a.button.ajax_add_to_cart_button.btn.btn-default";
-        this.proceedToCheckoutButton = "[style*='display: block;'] .button-container > a";
+        // Just get a span that's inside in a ajax_add_to_cart class button.
+        this.addToChartButton = ".ajax_add_to_cart_button span";
+        // Get the a label that's inside in a label with layer_cart id.
+        this.proceedToCheckoutButton = "#layer_cart a";
```

Comment:
for these two you can use an attribute to specify that we are looking for the add to cart and the proceed to checkout button